PluginCore/PluginCore/Managers/ShortcutKeysManager.cs

```diff
@@ -0,0 +1,371 @@
+﻿using System;
+using System.Collections;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+using PluginCore.Controls;
+
+namespace PluginCore.Managers
+{
+    /// <summary>
+    /// A static manager class for advanced shortcut keys.
+    /// </summary>
+    public static class ShortcutKeysManager
```

Comment:
@elsassph This is the one I was talking about. This class uses `Reflection` frequently. Would there be performance issues? If so, would using the Expression API instead solve it?